### PR TITLE
[4.0] Actions logs toolbar

### PR DESCRIPTION
**Pull request by brianteeman**
All components only have the options and help on the right hand side this PR brings the action logs component into sync

### Before
image

### After
image

### CODE REVIEW

File: administrator/components/com_actionlogs/views/actionlogs/view.html.php [hunk 1]
```diff
@@ -98,12 +98,12 @@ protected function addToolbar()
 	{
 		JToolbarHelper::title(JText::_('COM_ACTIONLOGS_MANAGER_USERLOGS'), 'list-2');
 
-		JToolbarHelper::deleteList('JGLOBAL_CONFIRM_DELETE', 'actionlogs.delete');
+		JToolBarHelper::custom('actionlogs.exportSelectedLogs', 'download', '', 'COM_ACTIONLOGS_EXPORT_CSV', true);
+		JToolBarHelper::custom('actionlogs.exportLogs', 'download', '', 'COM_ACTIONLOGS_EXPORT_ALL_CSV', false);		JToolbarHelper::deleteList('JGLOBAL_CONFIRM_DELETE', 'actionlogs.delete');
```
Comment (Quy) [inline administrator/components/com_actionlogs/views/actionlogs/view.html.php:102]:
```suggestion
		JToolBarHelper::custom('actionlogs.exportLogs', 'download', '', 'COM_ACTIONLOGS_EXPORT_ALL_CSV', false);
		JToolbarHelper::deleteList('JGLOBAL_CONFIRM_DELETE', 'actionlogs.delete');
```

File: administrator/components/com_actionlogs/views/actionlogs/view.html.php [hunk 2]
```diff
@@ -98,12 +98,12 @@ protected function addToolbar()
...
 		$bar = JToolbar::getInstance('toolbar');
 		$bar->appendButton('Confirm', 'COM_ACTIONLOGS_PURGE_CONFIRM', 'delete', 'COM_ACTIONLOGS_TOOLBAR_PURGE', 'actionlogs.purge', false);
 		JToolbarHelper::preferences('com_actionlogs');
 		JToolbarHelper::help('JHELP_COMPONENTS_ACTIONLOGS');
-		JToolBarHelper::custom('actionlogs.exportSelectedLogs', 'download', '', 'COM_ACTIONLOGS_EXPORT_CSV', true);
-		JToolBarHelper::custom('actionlogs.exportLogs', 'download', '', 'COM_ACTIONLOGS_EXPORT_ALL_CSV', false);
+
```
Comment (Quy) [inline administrator/components/com_actionlogs/views/actionlogs/view.html.php]:
```suggestion
```

File: administrator/components/com_actionlogs/views/actionlogs/view.html.php (hunk 1 above)
Comment (brianteeman) [reply administrator/components/com_actionlogs/views/actionlogs/view.html.php:102]: No this was deliberate to put the delete actions at the end of the list see Banners/tracks as an example

**Comment by chmst** [comment]
You could 

use Joomla\CMS\Toolbar\Toolbar; 

for Toolbar like in other views.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24562">issues.joomla.org/tracker/joomla-cms/24562</a>.</sub>

**Comment by brianteeman** [comment]
I deliberately didn't do any namespacing here as it makes sense to do all of that in its own pr. Single purpose pr are much easier for people to test.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 4236002e7a7ffadd124c8373fe45c0cbd032912f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24562">issues.joomla.org/tracker/joomla-cms/24562</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 4236002e7a7ffadd124c8373fe45c0cbd032912f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24562">issues.joomla.org/tracker/joomla-cms/24562</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

File: administrator/components/com_actionlogs/views/actionlogs/view.html.php (hunk 1 above)
Comment (Quy) [reply administrator/components/com_actionlogs/views/actionlogs/view.html.php:102]: coding style: This is only moving `JToolbarHelper::deleteList('JGLOBAL_CONFIRM_DELETE', 'actionlogs.delete');` to the next line and not be on the same line 102.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**